addon/components/bs-alert.js

```diff
@@ -200,21 +194,21 @@ export default class Alert extends Component {
         this,
         function () {
           if (!this.isDestroyed) {
-            this.hidden = this._hidden = true;
+            this.hidden = true;
             this.args.onDismissed?.();
           }
         },
         this.fadeDuration,
       );
     } else {
-      this.hidden = this._hidden = true;
+      this.hidden = true;
```

Comment:
I'm surprised that this is not causing a modification after consumption error. The property is read earlier in the same method. I assume all complexity before was only about preventing the modification after consumption. Maybe timing changed and it's not invoked anymore during a rendering cycle? 🤔